libs/ledger-live-common/src/wallet-api/react.ts

```diff
@@ -861,7 +861,7 @@ export function useLocalLiveApp([LocalLiveAppDb, setState]: LocalLiveAppDB): Loc
   const addLocalManifest = useCallback(
     (newLocalManifest: LiveAppManifest) => {
       setState(discoverDB => {
-        const newLocalLiveAppList = discoverDB.localLiveApp.filter(
```

Comment:
add useEffect to init the state when no existing cache client side

```suggestion
  useEffect(() => {
    if (LocalLiveAppDb === undefined) {
      setState(discoverDB => {
        return { ...discoverDB, localLiveApp: INITIAL_PLATFORM_STATE.localLiveApp };
      });
    }
  }, [LocalLiveAppDb, setState]);
```